docs/pages/stream.rst

```diff
@@ -1,7 +1,118 @@
 .. _stream:
 
-Stream API
-==========
+V1 Stream API
+=============
+
+The :class:`~runeq.stream` module contains classes that fetch timeseries data using
+`V1 Stream API <https://docs.runelabs.io/stream/index.html>`_.
+
+
+The V1 API is supported, but is not longer under active development. We recommend
```

Comment:
I appreciate the wording to steer them away from v1. Will this not be released until all data (events/spans, Percept, etc.) are available from v2? Or are they going to read through this language to do something we're actually directing them to do? And maybe not a question for here, but I don't even know if we'll ever get Summit RC+S into v2...

Reply:
We are planning to release this now -- but you're right, there is some data that is still only available through the V1 APIs. I'll tweak this in the follow up PR